README.md

```diff
@@ -68,9 +76,35 @@ To compile the contracts:
 ./go contracts:compile
 ```
 
-## Testing
+---
+### Run
+To deploy instances of the contracts for local development without prior knowledge Ganache and Truffle, run the following command:
+```shell
+./go contracts:run
+```
+
+This command starts up Ganache on a random port and migrates all contracts to the Ganache instance. The Ganache instance will remain running the the background, even though the command prompt is available.
```

Comment:
"even though the command prompt is available" should be "even though the command prompt is not available"?

Reply:
No. Normally when you startup ganache (unless you specify it should run in the background and/or nohup mode),  there is no command prompt because the process is running in the foreground. When you execute the` ./go contracts:run` command, it starts up Ganache in the background and the command prompt returns. You can't see the process running. There's no indication it is running unless you run the `ps - ef | grep ganache` command

Reply:
OK.
I've missunderstood the sentence, thinking that the mentionned command prompt was the command prompt of Ganache-Cli...
Don't you think rephrasing 'event through the command prompt returns" instead of "is available" would help preventing the confusion?

Reply:
Changed to "returns".